app/utils.py

```diff
@@ -51,3 +53,44 @@ def geocode(address):
     else:
         coords = r.json()['results'][0]['geometry']['location']
         return coords['lat'], coords['lng']
+
+
+def get_current_weather(location):
+    """TODO write docstring"""
```

Comment:
oh, as in - are temperature, pressure, and humidity the conditions that are important to the cac? i think temperature is definitely, but unsure about the others. might want to check with them what is and is not important. in any case this is more wrt product design than the code itself.